node/npm/glide/index.ts

```diff
@@ -17,7 +18,9 @@ function loadNativeBinding() {
                     nativeBinding = require("@scope/glide-for-redis-linux-x64");
                     break;
                 case "arm64":
-                    nativeBinding = require("@scope/glide-for-redis-linux-arm64");
+                    nativeBinding = existsSync("./node_modules/@scope/glide-for-redis-linux-musl-arm64")
```

Comment:
I suppose this could've been done as the check for any platform

Reply:
We should have a different check to verify if we're running on musl, to avoid scenarios where we run on musl but glide-for-redis-linux-musl-X wasn't properly installed and therefore we're trying to install glide-for-redis-linux-arm64 instead of existing with the proper error.
Have you seen this thread for different methods to check if we're running on musl?
https://github.com/nodejs/node/issues/48204
We don't need it to be very performant as this is being called only once when the library is being installed.